Formula/haskell-stack.rb

```diff
@@ -26,7 +27,16 @@ def install
     # Reported 25 May 2016: https://github.com/commercialhaskell/stack/issues/2192
     (buildpath/"cabal.config").write("allow-newer: base,transformers\n")
 
-    install_cabal_package
+    cabal_sandbox do
+      cabal_install
+
+      # Let `stack` handle its own parallelization
```

Comment:
Yeah, we may not need to pass `-j#{jobs}` at all either, but it seems that's probably the "right" thing to do, as it's the same thing we do with cabal: https://github.com/Homebrew/brew/blob/master/Library/Homebrew/language/haskell.rb#L58